mailpoet/assets/js/src/settings/pages/key_activation/key_activation.tsx

```diff
@@ -147,7 +152,53 @@ export function KeyActivation() {
       <Label
         htmlFor="mailpoet_premium_key"
         title={t('premiumTabActivationKeyLabel')}
-        description={t('premiumTabDescription')}
+        description={
+          <>
+            {ReactStringReplace(
+              t('premiumTabDescription'),
+              /\[link\](.*?)\[\/link\]/g,
+              (text) => (
+                <a
+                  href="https://account.mailpoet.com/account?utm_source=plugin&utm_medium=settings&utm_campaign=activate-existing-plan&ref=settings-key-activation"
+                  target="_blank"
+                  rel="noopener noreferrer"
+                >
+                  {text}
+                </a>
+              ),
+            )}
+            <br />
+            <br />
+            {ReactStringReplace(
```

Comment:
I was wondering if assigning these two regex replacements to a constant outside of the render cycle and simply using them would be better, what do you?  These first replacements are not bound to any `prop` so in practice they are simply a constant by nature and don't need to re-calculate them in each render.

Reply:
Fair point. I pushed a new commit implementing this.